views/partials/participant-unanticipated-problems-list.html.twig

```diff
@@ -0,0 +1,49 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Document Unanticipated Problems</h3>
```

Comment:
For read only, remove the word "Document" from the heading, since we are just listing the existing Unanticipated Problems.